linkerd/examples/k8s.yaml

```diff
@@ -0,0 +1,19 @@
+admin:
+  port: 9990
+
+namers:
+- kind: io.l5d.experimental.k8s
+  host: localhost
+  port: 443
+  tls: true
+  tlsWithoutValidation: true
+  authTokenFile: linkerd/examples/io.l5d.k8s/kube_token
```

Comment:
any reason to have this file outside of acceptance-test.yaml?
if it's specific to a local kube setup, maybe document that